front/style/layout/_forms.scss

```diff
@@ -559,11 +564,12 @@ input[type=date]::-webkit-inner-spin-button {
 
         input[type="checkbox"] {
             width: 20px;
+            flex: none;
         }
 
         label {
             order: 2;
-            @include text--small;
+            // @include text--small;
```

Comment:
This comment is usefull ? can you remove this line ?

Reply:
Yes, I saw that was doing it it now 🙊